fineract-provider/src/main/resources/sql/migrations/core_db/V344__GSIM_migration_script.sql

```diff
@@ -0,0 +1,61 @@
+--
+-- Licensed to the Apache Software Foundation (ASF) under one
+-- or more contributor license agreements. See the NOTICE file
+-- distributed with this work for additional information
+-- regarding copyright ownership. The ASF licenses this file
+-- to you under the Apache License, Version 2.0 (the
+-- "License"); you may not use this file except in compliance
+-- with the License. You may obtain a copy of the License at
+--
+-- http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing,
+-- software distributed under the License is distributed on an
+-- "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+-- KIND, either express or implied. See the License for the
+-- specific language governing permissions and limitations
+-- under the License.
+--
+
+
+-- permissions added
+
+
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'CREATE_GSIMACCOUNT', 'GSIMACCOUNT', 'CREATE', 0);
+ 
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'APPROVE_GSIMACCOUNT', 'GSIMACCOUNT', 'APPROVE', 0);
+ 
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'ACTIVATE_GSIMACCOUNT', 'GSIMACCOUNT', 'ACTIVATE', 0);
+
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'APPROVALUNDO_GSIMACCOUNT', 'GSIMACCOUNT', 'APPROVALUNDO', 0);
+
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'UPDATE_GSIMACCOUNT', 'GSIMACCOUNT', 'UPDATE', 0);
+
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'REJECT_GSIMACCOUNT', 'GSIMACCOUNT', 'REJECT', 0);
+
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'DEPOSIT_GSIMACCOUNT', 'GSIMACCOUNT', 'DEPOSIT', 0);
+
+INSERT INTO `m_permission` (`grouping`, `code`, `entity_name`, `action_name`, `can_maker_checker`) VALUES ('portfolio', 'CLOSE_GSIMACCOUNT', 'GSIMACCOUNT', 'CLOSE', 0);
+
+
+-- new gsim table
+CREATE TABLE `gsim_accounts` (
+	`id` BIGINT(20) NOT NULL AUTO_INCREMENT,
+	`group_id` BIGINT(20) NOT NULL DEFAULT '0',
+	`account_number` VARCHAR(50) NOT NULL,
+	`parent_deposit` DECIMAL(19,6) NOT NULL DEFAULT '0.000000',
+	`child_accounts_count` INT(11) NOT NULL,
+	`accepting_child` TINYINT(4) NOT NULL DEFAULT '0',
+	`savings_status_id` SMALLINT(5) NOT NULL DEFAULT '0',
+	`application_id` DECIMAL(10,0) NULL DEFAULT '0',
+	PRIMARY KEY (`id`),
+	UNIQUE INDEX `gsim_account_no_UNIQUE` (`account_number`),
+	INDEX `FK_gsim_group_id` (`group_id`),
+	CONSTRAINT `FK_gsim_group_id` FOREIGN KEY (`group_id`) REFERENCES `m_group` (`id`)
+)
```

Comment:
migration script will break the application(application will not build ) since there is no semicolon after create table script. @avikganguly01  correct me if I am wrong.

Reply:
we ***REALLY*** need https://issues.apache.org/jira/browse/FINERACT-697 ... this kind of discussion should not even happen - PR should build fully automatically, and just fail.